src/config.js

```diff
@@ -13,6 +13,8 @@ const fs = require('fs');
 require('dotenv').config();
 const { from } = require('env-var');
 
+console.log(process.env);
```

Comment:
Please make it clear to readers of the log what this output is...use a template string such as \`process environment: ${util.inspect(process.env)}\`.

Reply:
Error from my side. This line was added when I debugged/stepped through the code. Not meant to remain in the code . I have removed it .